server/tests/participant.service.test.js

```diff
@@ -1023,4 +1026,44 @@ describe('Participants Service', () => {
       true
     );
   });
+
+  it('should returns post hire statuses', async () => {
+    await closeDB();
+    await startDB();
+    const employerAId = v4();
+
+    const participant1 = {
+      lastName: 'Extra',
+      firstName: 'Eddy',
+      phoneNumber: '2502223333',
+      emailAddress: 'eddy@example.com',
+      interested: 'yes',
+      nonHCAP: 'yes',
+      crcClear: 'yes',
+      preferredLocation: 'Fraser',
+      contactedDate: '09/09/2020',
+    };
+
+    const response = await makeParticipant(participant1);
+
+    // Set Participant Status post hire status
+    await createPostHireStatus({
+      participantId: response.id,
+      status: postHireStatuses.postSecondaryEducationCompleted,
+      data: {},
+    });
+
+    const participants = await getParticipants(
+      { isEmployer: true, id: employerAId, regions },
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
+      null,
```

Comment:
This feels like a bit of a code smell here. We may want to refactor the parameters of the getParticipants function.

Reply:
we may need to refactor getParticipants func. But this is not in the scope of this sprint.